text/vec-set.md

```diff
@@ -148,6 +151,6 @@ read is a big win. If you need to iterate over the data frequently, you may want
 
 It is always important that the weight associated with your dispatchables represent the actual time
 it takes to execute them. In this pallet, we have provided an upper bound on the size of the set,
-which places an upper bound on the computation - this means we can use constant weight annotations.
+which places an upper bound on the computation. This means we can use constant weight annotations.
```

Comment:
I prefer the dash here as opposed to starting a new sentence; I think it makes it more clear what "this" is referring to.